web/tests/reactions.test.js

```diff
@@ -1189,7 +1189,6 @@ test("error_handling", ({override, override_rewire}) => {
         emoji_code: "991",
         user_id: 99,
     };
-    override_rewire(reactions, "current_user_has_reacted_to_emoji", () => true);
```

Comment:
This function doesn't get called in this test anymore because of an added early return